pennylane/ops/op_math/adjoint.py

```diff
@@ -21,13 +21,15 @@
 from pennylane.operation import Observable, Operation, Operator
 from pennylane.queuing import QueuingManager
 from pennylane.tape import make_qscript
+from pennylane.compiler import compiler
+from pennylane.compiler.compiler import CompileError
 
 from .symbolicop import SymbolicOp
 
 
 # pylint: disable=no-member
 def adjoint(fn, lazy=True):
-    """Create the adjoint of an Operator or a function that applies the adjoint of the provided function.
+    """A :func:`~.qjit` compatible adjoint transformation that creates the adjoint of an Operator or a function
```

Comment:
I know it was repeated before but it stutters quite a bit imo
```suggestion
    """A :func:`~.qjit` compatible transformation that creates the adjoint of an Operator or a function
```